spec/integration/full_project_spec.rb

```diff
@@ -554,6 +555,35 @@
     cloned_project.delete
   end
 
+  it "should be able to export raw report" do
+    m = @project.facts.first.create_metric
+    metric_name = Dir::Tmpname.make_tmpname ['metric.some_metric'], nil
+    metric_name.delete!('-')
+    m.identifier = metric_name
+    m.save
+
+    commits_data = [
```

Comment:
Similar code found in full_project_spec.rb:101, full_project_spec.rb:165, full_project_spec.rb:565 (mass = 66)